packages/elements/src/slider/index.ts

```diff
@@ -706,20 +770,18 @@ export class Slider extends ControlElement {
       else if (distanceFrom > distanceTo) {
         this.activeThumb = this.thumbs[1];
       }
-      this.thumbs.forEach((el: HTMLElement) => {
-        el.style.zIndex = '5';
+
+      this.thumbs.forEach((thumb: HTMLElement) => {
+        thumb.style.zIndex = '5';
```

Comment:
Why `5`? :)

Reply:
From looking at the code, I think it is just to make thumb elevated above the `part=track-fill` any number more than `2` should be fine. I'll change to `3` for `thumbs` and `4` for `activeThumb`.